### DISCUSSION TITLE
Export or print to PDF

### DISCUSSION THREAD

**Discussion by humphreylee** [helix-editor/helix#6289] [Ideas] [answered]
There is a [pandoc ](https://github.com/helix-editor/helix/issues/1541) way to export to PDF, but I wonder if an integrated approach can be implemented to provide complete solution, like how other functionalities being baked into Helix, like typical modern GUI editor. Thanks for all the hard work.

**Comment by pascalkuthe** [comment, marked as answer]
I don't think it makes sense to include that in core. There is no reason to do this inside helix instead of using an external command (be it pandoc or something else). It doesn't integrate with editing in any specical way. It's just kind of text related.

While we do include more feature in core than something like vim. The goal is not to include anything that might be useful as some GUI programs do but primarily functionality that actually enhance s he text editing experience (like lsp) or closely interacts with it in some way. The line can be fuzzy sometimes but for me exporting to PDF clearly doesn't need to be in core. In the future thisncoukd be a plugin that calls pandoc (or something else) once anpmugin system lands.

**Comment by gabydd** [comment]
Agree with everything Pascal said, this should be a plugin (or maybe if the build server protocol gets implemented it could be used for this), anyways I think this should probably be converted to a discussion as it is not an actionable issue.

**Comment by humphreylee** [comment]
Agreed. Hope this will not be forgotten.

**Comment by YardQuit** [comment]
Meanwhile in Linux if you are working from text files, install a2ps and ps2pdf and from helix ':sh a2ps filename.txt -o filename.ps | ps2pdf filename.ps filename.pdf' the concept should work for more file-formats using pandoc and other converters.
